snyk-monitor/values.yaml

```diff
@@ -8,18 +8,25 @@ monitorSecrets: snyk-monitor
 certsConfigMap: snyk-monitor-certs
 registriesConfConfigMap: snyk-monitor-registries-conf
 
-# The ConfigMap to use for loading policies into snyk-monitor.
-workloadPoliciesMap: snyk-monitor-workload-policies
+# An external ConfigMap to use for loading policies into snyk-monitor.
+# When using this the default policy shouldn't be deployed, set `defaultWorkloadPolicy.enabled: false`.
+workloadPoliciesMap: ''
+
 # A list of Snyk Organization public IDs to let snyk-monitor know in which Organization to auto-import and auto-delete scanned images.
-policyOrgs:
+# This is used to populate the default workload policy file.
+policyOrgs: []
+
+# Default rego workload policies to install.
+# Setting `workloadPoliciesMap` will overwrite this.
+defaultWorkloadPoliciesMap: snyk-monitor-workload-policies
 
 # One of: Cluster, Namespaced
 # Cluster - creates a ClusterRole and ClusterRoleBinding with the ServiceAccount
 # Namespaced - creates a Role and RoleBinding with the ServiceAccount
 scope: Cluster
 
 # The endpoint that being used to transmit monitored information
-integrationApi: ""
+integrationApi: ''
```

Comment:
@richerve a tiny question, what is the difference between `''` and `""` in Helm world?

Reply:
Oh sorry, that was my linter working. This is just yaml, there's no real distinction between '' and "". In helm values.yaml is static, there's no interpolation whatsoever so the semantics of yaml remain. Do you want me to revert to `""`?

Reply:
@richerve I think like we've already asked you for a lot of changes 😄 
Just this change doesn't have anything to do with the initial issue and I would like to leave this change out if possible 🙏